test/end-to-end/offdays/offdays.page.js

```diff
@@ -17,7 +17,7 @@ class OffdayPage {
     await FU.buttons.create();
     await FU.input('OffdayModalCtrl.offday.label', offday.label);
 
-    components.dateEditor.set(new Date(offday.date), null, '[uib-modal] .title');
+    await components.dateEditor.set(offday.date, 'offday-date-editor');
```

Comment:
If anyone is wondering, the critical change was putting an `await` in front of the component.